ogham/exa
# use GitHub Actions instead of Travis CI

- Trigger CI only if necessary
- Test on Ubuntu and macOS
- For each one, test MRSV (1.48.0), stable, beta and nightly

In the future:
- Very easy to add Windows when it’ll be supported by exa
- See how to test other architectures (Linux arm64 and armv7; macOS arm64)
- See how to test musl builds
- See how to generate builds for all platforms and architectures
